doc/tests.rst

```diff
@@ -585,8 +585,9 @@ an entire test suite.
 The ``cram`` stanza accepts the following fields:
 
 - ``enabled_if`` - controls whether the tests are enabled
-- ``alias`` - alias that can be used to run the test (in addition to
-  ``runtest``)
+- ``alias`` - alias that can be used to run the test. In addition to the user
+  alias, every test ``foo.t`` is attached to the ``@runtest`` alias and gets its
+  own ``@foo`` alias to make it convenient to run individually.
```

Comment:
does that apply to file tests or directory tests? in the case of directory tests, does that register their entrypoint as well? (ie if I have feature.t/subcase.t, is this attached to `@feature` or `@subcase`?)

It seems a bit too wide to set up a global alias for this without namespacing. For example if `install.t` attaches itself to `@install` that could be a problem.